src/server/onboarding-router/schema.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+
+export const ZOnboardMutationSchema = z.object({
```

Comment:
Lets give them readable (self-explanatory names)

Eg: `ZodOnboardingMutationSchema` and `TrpcOnboardingMutationSchema`

Even if they are long, they are easier to read.